README.md

```diff
@@ -1,63 +1,101 @@
-# Maslo Companion App
+# Maslo Companion Kit
 
 
 Clone the respository and ensure you have the requirements below.
 
 ### Requirements
 
+ * React Native CLI development environment for both iOS and Android (instructions [`here`](https://reactnative.dev/docs/environment-setup))
+
  * Node.js 10 ([`nvm`](https://github.com/nvm-sh/nvm) is preferable)
 
  To install:
  ```nvm install 10```
- 
+
  To switch to version 10:
  ```nvm use 10```
+
  * Expo CLI
  ```
  npm i -g expo-cli
  ```
+ * React-native CLI
+ ```
+ npm i -g react-native-cli
+ ```
  * Firebase Tools:
  ```
  npm i -g firebase-tools
  ```
-### Clone external dependencies
 
-1. Create a folder inside of the ```mobile``` folder called ```dependencies``` and navigate inside it.
-2. Clone both [Maslo Persona](https://github.com/HeyMaslo/maslo-persona/tree/dev-ts) from the dev-ts branch and [React Native Switch Pro](https://github.com/HeyMaslo/react-native-switch-pro) from the master branch inside the ```mobile/dependencies``` folder.
-3. Now rename the ```maslo-persona``` folder that you cloned in the previous step to ```persona```.
 
 
-### Install dependencies and validate Node.js version
+## Configuration Setup
```

Comment:
Is this like a "step" in the set up process? (E.g. step 1 is "Requirements", step 2 is this, etc.)

If so, please number each of these headings and at the very top state that these are the steps required to get the application set up. As of right now, it's a bit confusing what these headings are and which sections are required for set up. Also, if these are steps, let's keep the heading wording consistent so they should all be starting with a verb e.g. "Install Requirements", "Configure the App", etc.

Reply:
Yes these are like steps!